core/src/main/java/com/g16/feyrune/controller/WorldInputProcessor.java

```diff
@@ -51,31 +68,71 @@ public boolean keyUp(int keycode) {
         }
         return true;    }
 
+    /**
+     * Does nothing
```

Comment:
Don't know if "does nothing" is actually the correct term for (all of) these , should be like "handles X". At least that way the comment won't easily become out of date.